llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1236,20 +1235,21 @@ class LoopVectorizationCostModel {
   /// the loop.
   void collectInstsToScalarize(ElementCount VF);
 
-  /// Collect Uniform and Scalar values for the given \p VF.
+  /// Collect Uniforms, Scalars, and Insts to Scalarize for the given \p VF.
```

Comment:
Would be good to clarify the comment to connect more to the name. 
```suggestion
  /// Collect values that will not be widened, including Uniforms, Scalars, and Instructions  to Scalarize for the given \p VF.
```